apps/builder/app/builder/features/topbar/share.tsx

```diff
@@ -37,8 +37,13 @@ export const ShareButton = ({ projectId }: { projectId: Project["id"] }) => {
           </FloatingPanelPopoverTrigger>
         </Tooltip>
       </FloatingPanelAnchor>
-
-      <FloatingPanelPopoverContent css={{ zIndex: theme.zIndices[1] }}>
+      <FloatingPanelPopoverContent
+        sideOffset={14}
```

Comment:
you can access the theme using rawTheme.spacing[8] though radix expects a number here without units, what we did so far is `parseFloat(rawTheme.spacing[8])` its not ideal though